geolocator/pubspec.yaml

```
@@ -26,8 +26,8 @@ dependencies:
   flutter:
     sdk: flutter
   
-  geolocator_platform_interface: ^4.0.0
-  geolocator_android: ^3.0.2
+  geolocator_platform_interface: ^4.0.0+1
```

Version 4.0.0+1 doesn't exists, this should probably be updated to version 4.0.3.
```suggestion
  geolocator_platform_interface: ^4.0.3
```